pysixdesk/lib/utils.py

```diff
@@ -56,6 +57,19 @@ def download_output(filenames, dest, zp=True):
                 shutil.copy(filename, dest)
 
 
+def check_fort3_block(fort3, block):
+    '''Check the existence of the given block in fort.3'''
+    if not os.path.isfile(fort3):
+        print("The file %s doesn't exist" % fort3)
+        return 0
+    with open(fort3, 'r') as f_in:
+        lines = f_in.readlines()
+    for line in lines:
+        if line.lower().startswith(block.lower()):
+            return 1
+    return 0
+
```

Comment:
For python3, it's totally same !

Reply:
In python 2 also 0 and 1 get automatically converted to bool True and False when in a logical expression. But why have to the extra step of conversion (even if it is automatic) when there is no reason for it ?

Reply:
Because for me, it really doesn't matter with these tiny things. It will not influence anything, just more comfortable when coding and typing. :）
If you think it's necessary, just change it, it's totally fine with me.